app/screens/App.js

```diff
@@ -5,17 +5,11 @@ import Colors from '../shared/styles/Colors'
 registerScreens()
 
 const navigatorStyle = {
-  screenBackgroundColor: '#000000',
-  drawUnderNavBar: true,
```

Comment:
All of these seemed like extra. I tested them all and they didn't affect the styling to be different from what we need them to be.